packages/amplication-client/src/VersionControl/CommitResourceListItem.tsx

```diff
@@ -31,54 +31,51 @@ const CommitResourceListItem = ({ build }: Props) => {
   }, [build.commitId, build.resourceId, commitChangesByResource]);
 
   return (
-    <Panel className={CLASS_NAME} panelStyle={EnumPanelStyle.Bordered}>
-      <div className={`${CLASS_NAME}__row`}>
-        {build && build.resource && (
-          <>
-            <div className={`${CLASS_NAME}__title`}>
-              {build.resource && (
-                <ResourceCircleBadge type={build.resource.resourceType} />
-              )}
-              <p>{build.resource?.name}</p>
+    
+      
+        <NavLink
+          to={`/${currentWorkspace?.id}/${currentProject?.id}/${build?.resourceId}`}
+        >
+          {build && build.resource && (
+          <Panel
+            className={CLASS_NAME}
+            clickable
+            panelStyle={EnumPanelStyle.Bordered}
+          >
+            <div className={`${CLASS_NAME}__row`}>
+              <ResourceCircleBadge type={build.resource.resourceType} />
+
+              <span className={`${CLASS_NAME}__title`}>
+                {build.resource.name}
+              </span>
+
+              <span className="spacer" />
               <Link
-                to={`/${currentWorkspace?.id}/${currentProject?.id}/${build.resourceId}`}
```

Comment:
I think we should keep the link to the resource somehow. I see that it is not included in the new design. 
@alexbass86 please let me know what you think

Reply:
The link for the resource is there. Instead of an icon that holds the link, the whole card is the link

Reply:
You can see it on the video I attached to the PR description

Reply:
The idea was that a click on the card would link to the log (build page) - the user is looking on a list of builds - the default action should be to navigate to the build, not to the service